ledger/notary-plugin/src/main/java/net/corda/v5/ledger/notary/plugin/core/NotaryException.java

```diff
@@ -0,0 +1,58 @@
+package net.corda.v5.ledger.notary.plugin.core;
+
+import net.corda.v5.base.annotations.CordaSerializable;
+import net.corda.v5.base.exceptions.CordaRuntimeException;
+import net.corda.v5.crypto.SecureHash;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Representation of errors that can be returned by the notary (plugin). This is only a marker interface, the plugins
+ * can define their own errors by implementing this interface. Please refer to the non-validating notary plugin for
+ * a more detailed example.
```

Comment:
Looks like a copy / paste from the `NotaryError` file, but it doesn't make sense - this isn't an interface. You probably need to start from scratch given the change in structure.